### PR TITLE
Add process environment for FreeBSD

### PR DESCRIPTION
**Pull request by overhacked**
Relatively simple change to add ability to view process environment under FreeBSD. Only works when running as root.

### CODE REVIEW

File: freebsd/Platform.c [hunk 1]
```diff
@@ -220,9 +221,30 @@ void Platform_setZfsCompressedArcValues(Meter* this) {
...
+   int mib[4];
+   char *env = NULL;
+
+   mib[0] = CTL_KERN;
+   mib[1] = KERN_PROC;
+   mib[2] = KERN_PROC_ENV;
+   mib[3] = pid;
+
+   size_t capacity = ARG_MAX;
+   env = xMalloc(capacity);
```
Comment (cgzones) [inline freebsd/Platform.c:233]: No need to declare `env` previously, just do it here

File: freebsd/Platform.c [hunk 2]
```diff
@@ -220,9 +221,30 @@ void Platform_setZfsCompressedArcValues(Meter* this) {
 }
 
 char* Platform_getProcessEnv(pid_t pid) {
-   (void) pid;
-   // TODO
-   return NULL;
+   int mib[4];
```
Comment (cgzones) [inline freebsd/Platform.c:224]: maybe simply initialize with `int mib[4] = { CTL_KERN, KERN_PROC, KERN_PROC_ENV, pid }`

**Review by cgzones — COMMENTED** [review]
LGTM otherwise

**Comment by fasterit** [comment]
Works on FreeBSD 12.1 nicely. Thank you. Good to merge.

**Comment by cgzones** [comment]
Thanks, applied.

**Pull request merged**